src/components/RemsInterface/RemsInterface.jsx

```diff
@@ -56,44 +56,40 @@ export default class RemsInterface extends Component {
       console.log(metReq);
       return (
         <div>
-          <div className={"resource-entry"}>
-            <div className={"etasu-container"}>
-              <div className={"resource-entry-text"}  >{metReq.requirement.name}</div>
+          <div className={"resource-entry etasu-container"}>
+            <div className={"resource-entry-text"}  >{metReq.requirement.name}</div>
               <div className={"resource-entry-icon"}>{metReq.completed ? "✅"  : "❌"}</div>
-              <div className={"resource-entry-hover"}>{metReq.requirement.description}</div>
-            </div>
+            <div className={"resource-entry-hover"}>{metReq.requirement.description}</div>
           </div>
           {
-            metReq.childMetRequirements.map(subMetReq =>    
-              <div className={"resource-entry resource-child"}>
-                <div className={"etasu-container"}>
-                  <div className={"resource-entry-text"}>{subMetReq.requirement.name}</div>
-                  <div className={"resource-entry-icon"}>{subMetReq.completed ? "✅" : "❌"}</div>
-                  <div className={"resource-entry-hover"}>{subMetReq.requirement.description}</div>
-                </div>
+            metReq.childMetRequirements.map(subMetReq =>
+              <div className={"resource-entry resource-child etasu-container"}>
+                <div className={"resource-entry-text"}>{subMetReq.requirement.name}</div>
+                <div className={"resource-entry-icon"}>{subMetReq.completed ? "✅" : "❌"}</div>
+                <div className={"resource-entry-hover"}>{subMetReq.requirement.description}</div>
               </div>
             )
           }
         </div>
       )
     });
-      
-    }
-    // if (jsonData) {
-    //   return Object.keys(jsonData).map(element => {
-    //     console.log(element);
-    //     return (
-    //       // <div id={elementKey} className="jsonData" key={element} style={divStyle}>
-    //       //   <span className="elementKey">{element}</span>: <span className="elementBody">{jsonData[element] === null ? "null" : typeof jsonData[element] === "object" ? this.unfurlJson(jsonData[element], level + 1) : jsonData[element]}</span>
-    //       // </div>
-    //       <div>
-    //         <div className={"resource-entry"}>
-    //             <div>TEST</div>
-    //         </div>
-    //       </div>
-    //     )
-    //   });
-    // }
+
+  }
```

Comment:
is this test code?

Reply:
Dont think it was originally from me but got carried over. Deleted unnecessary commented code and improved styling/ formatting.